<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-IBVEUFH3" failures="1" tests="4" name="qafactory.swaglabs.testCases.TC001_Login_Functionality" time="24.099" errors="0" timestamp="18 Jul 2024 17:45:48 GMT">
  <testcase classname="qafactory.swaglabs.testCases.TC001_Login_Functionality" name="TC04_Validate_RemoveButton" time="3.259"/>
  <testcase classname="qafactory.swaglabs.testCases.TC001_Login_Functionality" name="TC02_Validate_Logout" time="2.011"/>
  <testcase classname="qafactory.swaglabs.testCases.TC001_Login_Functionality" name="TC03_Validate_About" time="10.737">
    <failure message="expected [The world relies on your code. Test on thousands of different device, browser, and OS configurations–anywhere, any time.] but found [[[EdgeDriver: MicrosoftEdge on windows (9578044749bb3b17581ec97eb441eece)] -&gt; xpath: //*[@id=&quot;__next&quot;]/div[2]/div[1]/div/div/div[1]/div/div[3]/p]]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [The world relies on your code. Test on thousands of different device, browser, and OS configurations–anywhere, any time.] but found [[[EdgeDriver: MicrosoftEdge on windows (9578044749bb3b17581ec97eb441eece)] -> xpath: //*[@id="__next"]/div[2]/div[1]/div/div/div[1]/div/div[3]/p]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:179)
	at qafactory.swaglabs.pageObjects.HomePage.validateMsg(HomePage.java:64)
	at qafactory.swaglabs.testCases.TC001_Login_Functionality.TC03_Validate_About(TC001_Login_Functionality.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- TC03_Validate_About -->
  <testcase classname="qafactory.swaglabs.testCases.TC001_Login_Functionality" name="TC01_Validate_Login" time="8.092"/>
</testsuite> <!-- qafactory.swaglabs.testCases.TC001_Login_Functionality -->
